client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -29,10 +31,25 @@ const DomainRegistrationSuggestion = React.createClass( {
 			isAdded = hasDomainInCart( this.props.cart, suggestion.domain_name ),
 			buttonClasses,
 			buttonContent,
-			domainFlag = null;
+			domainFlags = [],
+			translate = this.props.translate;
 
 		if ( suggestion.domain_name ) {
-			domainFlag = <DomainSuggestionFlag domain={ suggestion.domain_name }/>;
+			const newTLDs = [];
+
+			if ( newTLDs.some(
+					( tld ) => endsWith( suggestion.domain_name, tld ) && suggestion.domain_name.substring( 0, suggestion.domain_name.length - ( tld.length + 1 ) ).indexOf( '.' ) === -1
+				) ) {
+				domainFlags.push( <DomainSuggestionFlag content={ translate( 'New', { context: 'Domain suggestion flag' } ) }/> );
+			}
+		}
+
+		if ( suggestion.isRecommended ) {
+			domainFlags.push( <DomainSuggestionFlag content={ translate( 'Recommended', { context: 'Domain suggestion flag' } ) }/> );
+		}
+
+		if ( suggestion.isBestAlternative ) {
+			domainFlags.push( <DomainSuggestionFlag content={ translate( 'Best Alternative', { context: 'Domain suggestion flag' } ) }/> );
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Best Alternative' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.